zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/cache/ProcessCache.java

```diff
@@ -38,18 +40,22 @@
  */
 public class ProcessCache {
 
+  public static final String NAMESPACE = "zeebe.gateway.rest.cache";
```

Comment:
💭 I guess the namespace should not start with `zeebe`since this is the general Camunda REST gateway. Internally, it's still in the `zeebe` module but that is an implementation detail we should not expose in the metrics, maybe. The Camunda Exporter (cache) metrics should also ditch the `zeebe` prefix, I guess. But I cannot judge where this is used and if this breaks anything if we change it. Maybe @ChrisKujawa knows?

Reply:
We have a lot of old metrics that start with zeebe, historic related. 

I agree it makes sense for new metrics to start with camunda instead.

\cc @npepinpe

Reply:
Woule be great if this will be added to our dashboard as well

Reply:
We could do it together with https://github.com/camunda/camunda/issues/17442